provider/pkg/gen/schema.go

```diff
@@ -257,7 +257,7 @@ Use the navigation below to see detailed documentation for each of the supported
 	})
 	pkg.Language["python"] = rawMessage(map[string]interface{}{
 		"requires": map[string]string{
-			"pulumi":   ">=2.11.2,<3.0.0",
+			"pulumi":   ">=2.15.0,<3.0.0",
 			"requests": ">=2.21,<3.0",
 			"pyyaml":   ">=5.3.1,<5.4",
```

Comment:
Should we add an explicit requirement on `"semver": ">=2.8.1"`? [`pulumi` depends on](https://github.com/pulumi/pulumi/blob/260620430c4c8a41a8d7adfbd66f68ba6fd1e168/sdk/python/lib/setup.py#L41), so it will be pulled down as a dependency of `pulumi`, but technically we do make use of `semver` inside this provider in the generated `_utilities.py`, so I think we should list it here as a requirement. Note that the TF-based providers do include it as a requirement, e.g. [aws](https://github.com/pulumi/pulumi-aws/blob/f0267a7203da7ee233e5bba00767543a007d0a61/sdk/python/setup.py#L57).

Reply:
Looked into this, and semver is already handled by the codegen lib:
https://github.com/pulumi/pulumi/blob/260620430c4c8a41a8d7adfbd66f68ba6fd1e168/pkg/codegen/python/gen.go#L1582-L1585
https://github.com/pulumi/pulumi-kubernetes/blob/7aee6627f460333615fb75a9c6b37fbf51e5a09e/sdk/python/setup.py#L54-L60